src/main/java/lotto/service/LottoResultAdapter.java

```diff
@@ -0,0 +1,28 @@
+package lotto.service;
+
+import java.util.List;
+
+public class LottoResultAdapter implements Comparable<LottoResultAdapter>{
+
+    private static final int THREE = 3;
+    private final int matchCount;
+    private final List<Integer> myLottos;
+
+    public LottoResultAdapter(final int matchCount, final List<Integer> myLottos){
+        this.matchCount = matchCount;
+        this.myLottos = myLottos;
+    }
+
+    public boolean isPrize(){
+        return this.matchCount >= THREE;
+    }
+
+    public boolean isRateByRate(final int rate){
```

Comment:
이 메소드 명이 이해가 잘 안가서..matchCount랑 rate가 같은지 체크하는 boolean 함수라면,
```suggestion
    public boolean sameRate(final int rate){
```
같은 걸로 쓸 수 있지 않을까요?

아 그리고 rate 명이 이해가 잘 안가서..matchCount랑 rate가 같은지 체크하는 boolean 함수라면,
```suggestion
    public boolean sameRate(final int rate){
```
같은 걸로 쓸 수 있지 않을까요?

아 그리고 rate보단 더 명확한 단어를 사용하는게 좋을꺼 같아요 ㅜ rate는 통계, 비율같은 단위와 관련된 용어같아서..